src/main/resources/sql/dev/db-data-city-postcode.sql

```diff
@@ -0,0 +1,25 @@
+DELETE FROM CityPostcode ;
```

Comment:
The folder sql in the resources is redundant. With flyway we don't need it any more